tcms/bugs/tests/test_views.py

```diff
@@ -209,3 +209,65 @@ def test_record_changes(self):
             comments.last().comment,
             "Summary: %s -> %s\n" % (old_summary, new_summary)
         )
+
+
+class TestAddComment(LoggedInTestCase):
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+        user_should_have_perm(cls.tester, 'django_comments.add_comment')
+        user_should_have_perm(cls.tester, 'bugs.view_bug')
+
+        cls.bug = BugFactory()
+
+        cls.url = reverse('bugs-comment')
+        cls.post_data = {
+            'bug': cls.bug.pk,
+            'text': 'A comment text',
+        }
+
+    def test_add_comment(self):
+        old_comment_count = get_comments(self.bug).count()
+
+        response = self.client.post(self.url, self.post_data, follow=True)
+        comments = get_comments(self.bug)
+
+        self.assertRedirects(
+            response,
+            reverse('bugs-get', args=(self.bug.pk,)),
+            status_code=302,
+            target_status_code=200
+        )
+
+        self.assertEqual(comments.count(), old_comment_count + 1)
+        self.assertEqual(
+            comments.last().comment,
+            'A comment text'
+        )
```

Comment:
This should be part of the permissions test scenario b/c otherwise the scenario doesn't distinguish between a successfully added comment or a page redirect with errors.